book.rb

```diff
@@ -0,0 +1,10 @@
+class Book
```

Comment:
- [ Required ] It would be better to set up an `add_rental` method to the `Book` class to set up the `A Book` has `many rentals` association. This method can be used anytime to show the `Rental` records of a book, You can achieve this by adding the method below in the `Book` class.

```ruby
  def add_rental(person, date)
    Rental.new(date, self, person)
  end
```